expat/PSPBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=expat
+pkgver=2.1.0
+pkgrel=1
+pkgdesc="XML parsing C library"
+arch=('mips')
+url="https://libexpat.github.io/"
+license=('MIT')
+depends=()
+makedepends=()
+optdepends=()
+source=(
+    "http://sourceforge.net/projects/expat/files/${pkgname}/${pkgver}/${pkgname}-${pkgver}.tar.gz"
+    "config.sub"
```

Comment:
I think we should not really include this config sub everywhere.
Maybe add it as external source file?
`https://raw.githubusercontent.com/pspdev/psplibraries/c4746da120f66f08e07d714334fcf557e259f6cc/patches/config.sub`

Reply:
Okay, this change removes over 3000 lines from this PR